src/main/kotlin/AddressConstants.kt

```diff
@@ -47,6 +47,11 @@ object AddressConstants {
    */
   const val CLUSTER_NODE_LEFT = "steep.cluster.nodeLeft"
 
+  /**
+   * Will be published when the application is ready to handle requests.
+   */
+  const val HTTP_ENDPOINT_APPLICATION_READY = "steep.httpEndpoint.applicationReady"
```

Comment:
This message is not necessary since the HttpEndpoint is always the last verticle to boot up (except for the Garbage Collector which can run in the background and does not contribute to the application's health and, as a matter of fact, should not block the application from becoming ready)